DSCResources/MSFT_xRemoteFile/MSFT_xRemoteFile.psm1

```diff
@@ -253,7 +253,13 @@ function Test-TargetResource
                 }
             } else {
                 Write-Debug "MatchSource is false. No need for downloading file."
```

Comment:
Please move this message before line:
```
$fileExists = $true
```
and add another one saying that file will be downloaded if size isn't the same.